metc-wastewater-covid-monitor/server.R

```diff
@@ -287,16 +288,27 @@ server <- function(input, output) {
           buttons = c("copy", "excel", "csv"),
           searching = FALSE,
           lengthMenu = FALSE
-        )
+        ),
+        colnames = c("Date",
+                     "7-day average viral load in wastewater, M copies/person/day",
```

Comment:
is this correct that the load is a 7-day average, not a one-day measure? Cases are a 7-day average.

Reply:
Pretty sure it's a one day measure - if not, let's provide that over the 7 day aveage

Reply:
Yes! See data-wrangle.Rmd [here](https://github.com/Metropolitan-Council/covid-poops/blob/462e33902a3661ce419361caa417f8a93443566e/data-wrangle.Rmd#L340)

Reply:
this is an average of multiple samples, each day - not an average across a week